syntax = "proto3";

service SetCase {
  rpc UpperCase( stream Input) returns (stream Output);
  rpc LowerCase(stream Input) returns (stream Output);
}

message Input {
	string text_in = 1; 
}

message Output {
	string text_out  = 1;
}
